modules/libarchive/3.7.3/presubmit.yml

```diff
@@ -0,0 +1,27 @@
+bcr_test_module:
+  module_path: ""
+  matrix:
+    platform:
+      - debian10
+      - debian11
+      - ubuntu2004
+      # - ubuntu2004_arm64
+      - ubuntu2204
+      - macos
+      - macos_arm64
+      - windows
+    bazel: [6.x, 7.x, rolling]
+    build_flags:
+      - [--config=bcr_presubmit]
+      - [--config=bcr_presubmit, "--@libarchive//:use_mbedtls"]
```

Comment:
hmm, why does libarchive needs tls?

Reply:
oh for, crypto library...

Reply:
Yes. the config is not new, only diff here is that I'm actually getting the presubmit to run with it.

Reply:
makes sense